README.rst

```diff
@@ -103,61 +103,60 @@ Keyring Support
 ---------------
 
 Instead of typing in your password every time you upload a distribution, Twine
-allows you to store your username and password securely using `keyring`_.
-
-To use the keyring, you must first install the keyring packages:
+allows storing a username and password securely using `keyring`_.
+Keyring is installed with Twine but for some systems (Linux mainly) may
+require additional installation steps; refer to the keyring project page
+for details.
```

Comment:
Can this be a link to the “additional installation steps”?

Reply:
Done.